apps/server/src/modules/files/repo/sync-files.repo.ts

```diff
@@ -103,9 +103,9 @@ export class SyncFilesRepo {
 	async findFilesToSync(parentType: FileRecordParentType, aggregationSize: number): Promise<SyncFileItem[]> {
 		let itemDataList: SyncFileItemData[] = [];
 
-		if (parentType === FileRecordParentType.Task) {
+		if (parentType === FileRecordParentType.Submission) {
```

Comment:
Following my above comment, I would add an else-if-block here. 

For security reasons one could think about throwing an exception in the Task-block. But I would leave it. It would make it easy to add further blocks, like for grading. 

How did it work for lesson btw? Did you hardcode that too and remove it afterwards?

Reply:
We decided that level of generalization is enough for this code.